cartography/stats.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Needed this to add type hint on line 22 
See https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class